statsrelay.go

```diff
@@ -142,7 +169,7 @@ func sendPacket(buff []byte, target string, sendproto string, TCPtimeout time.Du
 			break
 		}
 		conn.Write(buff)
-		conn.Close()
+		defer conn.Close()
```

Comment:
This moves the Close() routine effectively to a go routine -- I imagine this has some speed benefits?